packages/mobile/src/features/videos/components/FiltersScreen/WhiteFilterButton.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
Czy Black i white filter button to kopie? czemu nie jest jeden komponent z parametrem decydujacym o tym czy jest aktywny?